mdpow/tests/test_analysis.py

```diff
@@ -65,9 +62,9 @@ def fep_benzene_directory(tmpdir_factory):
 class TestAnalyze(object):
     def get_Gsolv(self, pth):
         gsolv = pth.join("FEP", "water", "Gsolv.fep")
-        # Loading only works with magic code in restart.Journalled.load()
-        # so that Pickles produced with Python 2 can also be read with 3:
-        G = mdpow.fep.Ghyd(filename=gsolv.strpath)
+        with open(gsolv, 'rb') as f:
+            # Due issues with python 2 pickle files
+            G = pickle.load(f, encoding='latin1')
```

Comment:
```suggestion
        # Loading only works with magic code in restart.Journalled.load()
        # so that Pickles produced with Python 2 can also be read with 3:
        G = mdpow.fep.Ghyd(filename=gsolv.strpath)
```

The updated pickle reading code should take care of this and not need the special case here.